homework_4/bets_api/internal/api/controllers/validators/betRequestValidator.go

```diff
@@ -0,0 +1,18 @@
+package validators
+
+const lostStatus = "lost"
+const wonStatus = "won"
+const activeStatus = "active"
```

Comment:
FYI, this can also be defined as 
```suggestion
const (
    lostStatus = "lost"
    wonSatus = "won"
    activeStatus = "active"
)

Reply:
Love it, thanks!